### PR TITLE
Add abilty to configure postStart handler for teleport-cluster chart

### PR DESCRIPTION
**Pull request by stefansedich**
This PR introduces the ability to configure a `postStart` handler. We are using this to configure a script that will run `tctl create` of our roles and provider config when the server starts.

### CODE REVIEW

**Comment by russjones** [comment]
@webvictim Can you review?

**Review by webvictim — CHANGES_REQUESTED** [review]
@stefansedich Changes look good to me.

Please also add this value to the Helm chart reference at https://github.com/gravitational/teleport/blob/master/docs/pages/kubernetes-access/helm/reference.mdx so that it's kept up to date.

Thanks for the PR!

**Comment by stefansedich** [comment]
> @stefansedich Changes look good to me.
> 
> Please also add this value to the Helm chart reference at https://github.com/gravitational/teleport/blob/master/docs/pages/kubernetes-access/helm/reference.mdx so that it's kept up to date.
> 
> Thanks for the PR!

@webvictim that should be done.

**Review by webvictim — APPROVED** [review]

**Comment by webvictim** [comment]
@stefansedich Thanks - this will be merged once it has another approving review.

**Review by inertial-frame — APPROVED** [review]

File: docs/pages/kubernetes-access/helm/reference.mdx [hunk 1]
```diff
@@ -864,6 +864,29 @@ A list of `initContainers` which will be run before the main Teleport container
   </TabItem>
 </Tabs>
 
+## `postStart`
```
Comment (inertial-frame) [inline docs/pages/kubernetes-access/helm/reference.mdx:867]: Do you mind adding the same kind of table you added in https://github.com/gravitational/teleport/pull/7167/files:

```
## `extraEnv`

| Type | Default value | Can be used in `custom` mode? |
| - | - | - |
| `list` | `[]` | ✅  |
```

**Review by klizhentas — APPROVED** [review]

**Comment by stefansedich** [comment]
@klizhentas anything I need to do to get CI running?

**Comment by webvictim** [comment]
@stefansedich External PRs require approval to run, I just hit the start button.

**Pull request merged**